pkl-core/src/main/java/org/pkl/core/module/ModuleKeyFactories.java

```diff
@@ -79,7 +80,11 @@ public static ModuleKeyFactory classPath(ClassLoader classLoader) {
   }
 
   /**
-   * Returns a factory for external reader module keys.
+   * Returns a factory for external process module keys.
+   *
+   * <p>Using external process module keys requires a separate runtime dependency on <a
+   * href="https://github.com/msgpack/msgpack-java">msgpack-java</a> version 0.9.x, unless using
```

Comment:
Not a big fan of literal version numbers in doc-comments for risk of going out of sync.